event_model/tests/test_em.py

```diff
@@ -1196,3 +1196,47 @@ def test_pickle_filler():
     serialized = pickle.dumps(filler)
     deserialized = pickle.loads(serialized)
     assert filler == deserialized
+
+def test_projections():
+    event_model.schema_validators[event_model.DocumentNames.start].validate(
+    {
+        "uid": "abc",
+        "time": 0,
+        "projections": [
+            {
+                "name": "test",
+                "version": "42.0.0",
+                "configuration": {},
+                "projection": {
+                    'Entry/instrument/detector/data': {
```

Comment:
I am not sure we want to inherit the "full path" from nexus, but should be looking at what the keys in the `NXData` groups are. 

My understanding is that the `"insturment/..."` paths in nexus are directly analogous to the bodies of these projections and are still instrument / beam line / facility specific where as we want to map to the NXData definitions which re-map the devices specific mappings to a technique specific mapping.

Reply:
Right. But I was assuming that the job of this schema was to be able to project from places in the document model (stream/field/etc) to a uniquely-identified spot in a vocabulary, and though NeXuS is what is probably desired, we needed to be open.  My example in the test was a bit of a trial balloon on how the mapping might look for NeXuS,  but once projections are adopted in event_model, there will still have to be some sort of coordination about string-to-NeXuS field conventions. With NeXuS being XML-ish, I happened to slip into an XPath-ish syntax.

If we want to be more explicit about NeXuS pathing here, that would be more limiting but save some effort in coordination.